stable/prometheus-mysql-exporter/templates/secret-env.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "prometheus-mysql-exporter.fullname" . }}
+type: Opaque
+data:
+  DATA_SOURCE_NAME: "{{ .Values.mysql.user }}:{{ .Values.mysql.pass }}@{{ if .Values.mysql.protocol }}{{ .Values.mysql.protocol }}{{ end }}({{ .Values.mysql.host }}:{{ .Values.mysql.port }})/{{ if .Values.mysql.db }}{{ .Values.mysql.db }}{{ end }}{{ if .Values.mysql.param }}?{{ .Values.mysql.param }}{{ end }}"
```

Comment:
You need to find a way to base64 encode the secret, kubernetes will refuse it otherwise

Reply:
Have tried somthing like this:
(in _helper.tpl)
```
{{- define "prometheus-mysql-exporter.secret-data-source" -}}
{{ .Values.mysql.user }}:{{ .Values.mysql.pass }}@{{ if .Values.mysql.protocol }}{{ .Values.mysql.protocol }}{{ end }}({{ .Values.mysql.host }}:{{ .Values.mysql.port }})/{{ if .Values.mysql.db }}{{ .Values.mysql
.db }}{{ end }}{{ if .Values.mysql.param }}?{{ .Values.mysql.param }}{{ end }}
{{- end -}}
```

(in secret-env.yaml)
```
DATA_SOURCE_NAME: "{{ include "prometheus-mysql-exporter.secret-data-source" . | b64enc }}"
```

Reply:
Ah! Good catch! I will use `Secret`'s `stringData` if possible, or else pipe to `b64enc`. Let also figure out a better way to test this...

Reply:
As far as I can tell, `stringData` works as intended here 👍

Reply:
I'm not sure, but stringdata is something quite new in kubernetes? This could prevent some user to install? 

Looking at kubernetes documention, I find mention of stringdata from 1.12 and above. For exemple GKE (google kubernetes) still offers 1.11 clusters.

Don't know the policy of helm charts concerning version of kubernetes, but having to upgrade my cluster to install mysql-exporter, might be a good thing ;)